### DISCUSSION TITLE
Different results using regex101 and the core function "pattern"

### DISCUSSION THREAD

**Discussion by khband** [stoplightio/spectral#1744] [Q&A]
When I use the pattern ^\d+\.\d+\.\d+$ 

The core function "pattern" seems to allow "**2.0**" while regex101 says that it shouldn't match with that. Do you have any idea why this is happening? Thanks

**Comment by P0lip** [comment]
Hey!
My apologies for such a delayed response.
Could you please share the rule? There might be something off with the ecapes.
